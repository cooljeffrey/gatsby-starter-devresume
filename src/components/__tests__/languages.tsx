import React from "react";
import renderer from "react-test-renderer";
import Languages from "../languages";
describe("Languages", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<Languages items={[{ name: "English", level: "Native" }]} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
