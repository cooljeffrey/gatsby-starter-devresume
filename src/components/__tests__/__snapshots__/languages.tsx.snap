// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Languages renders correctly 1`] = `
<section
  className="skills-section py-3"
>
  <h3
    className="text-uppercase resume-section-heading mb-4"
  >
    Languages
  </h3>
  <ul
    className="list-unstyled resume-lang-list"
  >
    <li
      className="mb-2"
    >
      English
       
      <span
        className="text-muted"
      >
        (
        Native
        )
      </span>
    </li>
  </ul>
</section>
`;
